plugin.video.sito.tv/resources/lib/views.py

```diff
@@ -0,0 +1,358 @@
+# -*- coding: utf-8 -*-
+from resources.lib.sito import _lang
+from resources.lib.utils import *
+from resources.lib.api import *
+from kodiswift import xbmcgui, xbmc
+
+
+class ViewMode(object):
+    ListView = None
+    IconWall = 50
+
+
+def open_magnet(magnet_link):
+    log("open magnet: %s" % magnet_link)
+    if magnet_link:
+        try:
+            start_bitx(magnet_link)
+            kodi_go_back()
+            return
+        except Exception as ex:
+            log("BitX magnet start error: %s" % ex, level=xbmc.LOGERROR)
+
+    notice(_lang(33023), _lang(33022), 2500)
+    kodi_go_back()
+
+
+def kodi_go_back():
```

Comment:
why do you need this? Likely you are wrongly creating listitems, `isFolder=False` will not create browsable vfs directories